pkgs/development/go-modules/generic/default.nix

```diff
@@ -171,13 +177,24 @@ let
     buildPhase = args.buildPhase or ''
       runHook preBuild
 
+      exclude='\(/_\|examples\|Godeps\|testdata'
+      if [[ -n "$excludedPackages" ]]; then
+        IFS=' ' read -r -a excludedArr <<<$excludedPackages
+        if ((''${#excludedArr[@]} > 1)); then
+          printf -v excludedAlternates '%s\\|' "''${excludedArr[@]}"
+          excludedAlternates=''${excludedAlternates%\\|} # drop final \| added by printf
+          exclude+='\\|'"$excludedAlternates"
+        else
```

Comment:
Does this need an if/else or can we do this?
```
      exclude='\(/_\|examples\|Godeps\|testdata'
      if [[ -n "$excludedPackages" ]]; then
        IFS=' ' read -r -a excludedArr <<<$excludedPackages
        printf -v excludedAlternates '%s\\|' "''${excludedArr[@]}"
        excludedAlternates=''${excludedAlternates%\\|} # drop final \| added by printf
        exclude+='\\|'"$excludedAlternates"
      fi
      exclude+='\)'
```

Reply:
yes indeed, will update